kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/BrokerData.java

```diff
@@ -40,19 +45,20 @@ public BrokerData(
       @Nullable String rack,
       Relationship configurations,
       Relationship partitionReplicas) {
+    this.id = Objects.requireNonNull(id);
     this.links = Objects.requireNonNull(links);
     attributes = new Attributes(clusterId, brokerId, host, port, rack);
     relationships = new Relationships(configurations, partitionReplicas);
   }
 
   @JsonProperty("type")
   public String getType() {
-    return "KafkaBroker";
```

Comment:
This should still be `type: "KafkaBroker"` right? We were using this type of CamelCasing in ccloud world.

Reply:
Done.